example/lib/widgets/inbox_message_view_override.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:airship_flutter/airship_flutter.dart';
+import 'package:flutter/foundation.dart';
+
+class InboxMessageViewOverride extends InboxMessageView {
```

Comment:
We dont need to override this since its a static variable. I think adding something like:

```
// Uncomment to enable hybrid composition on Android
// InboxMessageView. hybridComposition = true
```

Reply:
Noted 👍